sonic-net/sonic-buildimage
# [Nokia][sonic-platform] Update nokia sonic-platform submodule with thermal coherence changes to be paired with NDK >= 22.9.26

These changes ensure proper thermal subsystem operation when removing ASIC internal thermal sensor polling from NDK.
     - Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316
    -  This PR is 202205 specific.  The associated master PR is here:  https://github.com/sonic-net/sonic-buildimage/pull/18564

Edit:  Now also includes kernel module changes for error level messages to warning level (this change is 202205 specific and not applicable to master) as well as fix for show chassis module with correct linecard description (https://github.com/Nokia-ION/ndk/issues/45).



#### Why I did it
These changes are required in order to ensure thermal subsystem coherence when removing 'bcmcmd show pvt' ASIC thermal sensor polling from NDK.

Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Made the necessary modifications to facilitate coherent temp sensor operations.

#### How to verify it
Verify that thermal subsystem operates normally via 'show platform temperature' command.

Please pair this submodule update with NDK >= 22.9.26 : NDK 22.9.26 PR is here:  https://github.com/Azure/sonic-buildimage-msft/pull/316



#### Which release branch to backport (provide reason below if selected)
